mautic/mautic
# plugin was not fetching the correct fields per object

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | na
| Related developer documentation PR URL |na 
| Issues addressed (#s or URLs) | na
| BC breaks? | na
| Deprecations? |na 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:


#### Steps to test this PR:
1. Apply this PR, run the fetch command for Salesforce
2. Match your fields - specifically boolean fields
3. Bug should not be displayed


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Enable your salesforce plugin
2. Select to match leads and contacts
3. run the fetch command 

`php app/console mautic:integration:fetchleads --integration=Salesforce --time-interval="29 days" -e=dev`
on dev mode for testing purposes. 

-An error is displayed when trying to fetch info from the contacts object.
